website/docs/reference/resource-configs/alias.md

```diff
@@ -4,10 +4,47 @@ description: "Read this guide to understand the alias configuration in dbt."
 datatype: string
```

Comment:
These are too far away from edited content to make a suggestion directly, but: 
- what does that `resource_types` array do? Does it configure how the docs render somehow? 
- Let's update the `description` to something like "Aliasing a resource allows it to be created in the database with a custom name, instead of relying on the filename"

Reply:
asked in the slack channel and updated desc